Application Actions/VLC/1.0/default.json

```diff
@@ -0,0 +1,11 @@
+{
+  "RotateLeft"	:   "Rewind",
+  "RotateRight"	:	"Fast Forward",
+  "ButtonPress"	:	"Play or Pause",
+  "SwipeUp"		:	"Increase Volume",
+  "SwipeDown"	:	"Decrease Volume",
+  "SwipeLeft"	:	"Previous",
+  "SwipeRight"	:	"Next",
+  "FlyLeft"		:	"Previous",
+  "FlyRight"	:	"Next"
+}
```

Comment:
Can you please properly format this file without tabs and left-aligning the actions? Thanks!